Python/Tests/GlassTests/PythonTests/Python/Python37.32.GlassTestProps

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<!-- This file defines the properties used for Python 3.6 32-bit debugging. -->
+<PropertyGroup xmlns="http://schemas.microsoft.com/vstudio/diagnostics/glasstestmanagement/2014">
+  <!--<PythonExe CopyToEnvironment="true">$(Registry:HKEY_LOCAL_MACHINE\SOFTWARE\Python\PythonCore\3.6\InstallPath@ExecutablePath)</PythonExe>-->
+  <!--<PythonExe CopyToEnvironment="true">$(Registry:HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Python\PythonCore\3.6-32\InstallPath@ExecutablePath)</PythonExe>-->
+  <!--<PythonExe CopyToEnvironment="true">C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python36_86\python.exe</PythonExe>-->
+  <PythonExe CopyToEnvironment="true">C:\Program Files (x86)\Python37-32\python.exe</PythonExe>
```

Comment:
This shouldn't be using hardcoded path to interpreter, since it's not guaranteed to be installed in that location. 

Was there something wrong in the commented-out code above, that uses the registry to retrieve the path of the default interpreter for that version?

Reply:
Last time (18 months ago), they didn't have a way of retrieving the path from the registry. While this may look like msbuild, it actually isn't. I had discussed it with them and there is a way that you can write your own 'plugin' so we could have implemented our own registry reader, but I didn't have time to do it. We have no choice but to use the default install path until then.